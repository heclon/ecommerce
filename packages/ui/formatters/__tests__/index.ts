import { formatters } from '../..'

describe('formatters', () => {
  describe('rrule', () => {
    test.each([[''], ['FREQ=DAILY;COUNT=1'], ['FREQ=WEEKLY;INTERVAL=1;BYDAY=MO,TU']])(
      'rrule formatter %s',
      (rrule?: string) => {
        expect(formatters.rrule(rrule)).toMatchSnapshot()
      }
    )
  })
})
