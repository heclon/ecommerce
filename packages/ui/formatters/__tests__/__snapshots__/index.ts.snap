// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatters rrule rrule formatter  1`] = `""`;

exports[`formatters rrule rrule formatter FREQ=DAILY;COUNT=1 1`] = `"One-off session"`;

exports[`formatters rrule rrule formatter FREQ=WEEKLY;INTERVAL=1;BYDAY=MO,TU 1`] = `"every week on Monday, Tuesday"`;
